$OpenBSD: patch-Makefile,v 1.2 2010/09/30 07:26:05 ajacoutot Exp $

XXX make it compile with gtk+2 2.22 (*_DISABLE_DEPRECATED)
gtk/gtk_tabs.c:41: error: expected declaration specifiers or '...' before 'GtkNotebookPage'

--- Makefile.orig	Mon Apr 19 22:56:28 2010
+++ Makefile	Thu Sep 30 07:49:36 2010
@@ -102,8 +102,8 @@ ifneq ($(TARGET),riscos)
   endif
 endif
 
-Q=@
-VQ=@
+Q=
+VQ=
 PERL=perl
 MKDIR=mkdir
 TOUCH=touch
@@ -424,7 +424,7 @@ endif
 # ----------------------------------------------------------------------------
 
 ifeq ($(TARGET),gtk)
-  LDFLAGS += $(shell $(PKG_CONFIG) --libs libxml-2.0 libcurl libhubbub openssl)
+  LDFLAGS += $(shell $(PKG_CONFIG) --libs libxml-2.0 libcurl libhubbub)
   LDFLAGS += $(shell $(PKG_CONFIG) --libs libcss)
 
   # define additional CFLAGS and LDFLAGS requirements for pkg-configed libs here
@@ -444,7 +444,6 @@ ifeq ($(TARGET),gtk)
   $(eval $(call pkg_config_find_and_add,PNG,libpng,PNG  ))
 
   GTKCFLAGS := -std=c99 -Dgtk -Dnsgtk \
-		-DGTK_DISABLE_DEPRECATED \
 		-D_BSD_SOURCE \
 		-D_XOPEN_SOURCE=600 \
 		-D_POSIX_C_SOURCE=200112L \
@@ -452,7 +451,7 @@ ifeq ($(TARGET),gtk)
 		-DGTK_RESPATH=\"$(NETSURF_GTK_RESOURCES)\" \
 		$(WARNFLAGS) -I. -g \
 		$(shell $(PKG_CONFIG) --cflags libglade-2.0 gtk+-2.0) \
-		$(shell $(PKG_CONFIG) --cflags libhubbub libcurl openssl) \
+		$(shell $(PKG_CONFIG) --cflags libhubbub libcurl) \
 		$(shell xml2-config --cflags)
 
   GTKLDFLAGS := $(shell $(PKG_CONFIG) --cflags --libs libglade-2.0 gtk+-2.0 gthread-2.0 gmodule-2.0 lcms)
@@ -799,26 +798,26 @@ clean: $(CLEANS)
 install-gtk: nsgtk
 	mkdir -p $(DESTDIR)$(NETSURF_GTK_RESOURCES)throbber
 	mkdir -p $(DESTDIR)$(NETSURF_GTK_BIN)
-	@cp -v nsgtk $(DESTDIR)$(NETSURF_GTK_BIN)netsurf
-	@cp -vRL gtk/res/adblock.css $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/arrow_down_8x32.png $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/blankpage $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/ca-bundle.txt $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/default.css $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/default.ico $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/favicon.png $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/gtkdefault.css $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/license $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/languages $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/netsurf.xpm $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/netsurf-16x16.xpm $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/quirks.css $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/themelist $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/throbber/*.png $(DESTDIR)$(NETSURF_GTK_RESOURCES)/throbber
-	@cp -vRL gtk/res/toolbarIndices $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/Aliases $(DESTDIR)$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/docs $(DESTDIR)/$(NETSURF_GTK_RESOURCES)
-	@cp -vRL gtk/res/SearchEngines $(DESTDIR)$(NETSURF_GTK_RESOURCES)SearchEngines
+	@cp nsgtk $(DESTDIR)$(NETSURF_GTK_BIN)netsurf
+	@cp -RL gtk/res/adblock.css $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/arrow_down_8x32.png $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/blankpage $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/ca-bundle.txt $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/default.css $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/default.ico $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/favicon.png $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/gtkdefault.css $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/license $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/languages $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/netsurf.xpm $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/netsurf-16x16.xpm $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/quirks.css $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/themelist $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/throbber/*.png $(DESTDIR)$(NETSURF_GTK_RESOURCES)/throbber
+	@cp -RL gtk/res/toolbarIndices $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/Aliases $(DESTDIR)$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/docs $(DESTDIR)/$(NETSURF_GTK_RESOURCES)
+	@cp -RL gtk/res/SearchEngines $(DESTDIR)$(NETSURF_GTK_RESOURCES)SearchEngines
 	gzip -9v < gtk/res/downloads.glade > $(DESTDIR)$(NETSURF_GTK_RESOURCES)downloads.glade
 	gzip -9v < gtk/res/history.glade > $(DESTDIR)$(NETSURF_GTK_RESOURCES)history.glade
 	gzip -9v < gtk/res/login.glade > $(DESTDIR)$(NETSURF_GTK_RESOURCES)login.glade
